source/production/F0.Minesweeper.Server/Pages/Index.razor.cs

```diff
@@ -1,9 +1,23 @@
+using F0.Minesweeper.Components;
 using F0.Minesweeper.Components.Abstractions;
+using F0.Minesweeper.Components.Abstractions.Enums;
 
 namespace F0.Minesweeper.Server.Pages
 {
 	public partial class Index
 	{
 		public MinefieldSize MinefieldSize = new MinefieldSize(10, 10);
+
+		public Minefield? Minefield1 { get; set; }
```

Comment:
I would just name it `Minefield`. Adding the one doesn't make sense right now since we don't support having multiple minefields at the time. But maybe that's a future game mode to be had? :D

Reply:
Yes!

Unintentional! Or better said, I have kept The One as my hidden "to-do"

We will move the selector to the Game component suggested below :)